FishjamClient/build.gradle

```diff
@@ -2,14 +2,14 @@ plugins {
     id 'com.android.library'
     id 'org.jetbrains.kotlin.android'
     id "org.jetbrains.dokka"
-    id "com.google.protobuf" version "0.9.3"
+    id "com.google.protobuf" version "0.9.4"
     id 'maven-publish'
 
     id "kotlin-android"
 }
 
 android {
-    namespace 'com.jellyfishdev.jellyfishclient'
+    namespace 'com.fishjam.client'
```

Comment:
as we are still using fishjam-dev org, maybe we can stick with `com.fishjamdev.client`?

Reply:
👍